cobol.html.markdown

```diff
@@ -0,0 +1,1016 @@
+---
+language: COBOL
+filename: learncbl.cbl
+contributors:
+    - ["Mikael Kemstedt", "https://github.com/MikaelKemstedt"]
+---
+
+COBOL stands for "**Co**mmon **B**usiness **O**riented **Language**.
+It is a high-level programming language first developed in 1959 
+(standardised in 1968) and as the name suggests it is designed for developing 
+business with mostly file-oriented applications.
+
+Today it's still in use mainly because there's just so much code written, and
```

Comment:
I think the sentence structure should be slightly changed:

"It's still in use today, mainly because there are so many applications implemented in COBOL and the cost of maintenance is often lower than migrating to another language"

I don't see any reason to mention Java here.